test/toxiproxy_test.rb

```diff
@@ -426,6 +426,25 @@ def test_invalid_direction
     end
   end
 
+  def test_toxiproxy_whitelists_webmock
+    WebMock.enable!
+    WebMock.disable_net_connect!
+    endpoint = "#{Toxiproxy.uri.host}:#{Toxiproxy.uri.port}"
+
+    WebMock::Config.instance.allow = nil
+    Toxiproxy.version # This should initialize the list.
+    assert WebMock::Config.instance.allow.include?(endpoint)
+
+    WebMock::Config.instance.allow = ['some-other-host']
```

Comment:
I'd split the test into two cases: 1) when `WebMock::Config.instance.allow` is nil and 2) when `WebMock::Config.instance.allow` has user-defineted hosts